service/history/timer_queue_active_task_executor_test.go

```diff
@@ -1727,6 +1727,9 @@ func (s *timerQueueActiveTaskExecutorSuite) TestWorkflowExecutionTimeout_Fire()
 	s.mockExecutionMgr.EXPECT().GetWorkflowExecution(gomock.Any(), gomock.Any()).Return(&persistence.GetWorkflowExecutionResponse{State: persistenceMutableState}, nil)
 	s.mockExecutionMgr.EXPECT().UpdateWorkflowExecution(gomock.Any(), gomock.Any()).Return(tests.UpdateWorkflowExecutionResponse, nil)
 
+	// advance the clock to be sure workflow is expired
```

Comment:
Curious why this was not needed before?

Reply:
that was a part of a bug - timeout task triggered before it should be, but still terminates the workflow.